specs/windows/drivers.table

```diff
@@ -1,5 +1,5 @@
 table_name("drivers")
-description("Lists all installed and loaded Windows Drivers and their relevant data")
+description("Lists all installed and loaded Windows Drivers and their relevant data.")
```

Comment:
Would you mind changing this to `Windows system drivers and their relevant data.`?